app/forms/journeys/further_education_payments/taught_at_least_one_term_form.rb

```diff
@@ -0,0 +1,37 @@
+module Journeys
+  module FurtherEducationPayments
+    class TaughtAtLeastOneTermForm < Form
+      attribute :taught_at_least_one_term, :boolean
+
+      validates :taught_at_least_one_term,
+        inclusion: {
+          in: [true, false],
+          message: ->(object, data) { i18n_error_message(:inclusion, school_name: object.school.name).call(object, data) }
```

Comment:
this was a bit quirky not 100% what the api should be but this seems to work and my tests pass